test-e2e/tests/account_test.ts

```diff
@@ -53,41 +45,53 @@ Scenario('I can see my account data', async ({ I }) => {
   I.see('Edit information');
 
   I.see('Terms & tracking');
-  I.see('I accept the Terms and Conditions of Cleeng.');
+  I.see(`I accept the Terms and Conditions of ${current.authProvider}.`);
   I.see(consentCheckbox);
 
   I.seeInCurrentUrl(constants.accountsUrl);
 });
 
-Scenario('I can cancel Edit account', async ({ I }) => {
+Data(configs).Scenario('I can cancel Edit account', async ({ I, current }) => {
+  if (!current.canEditEmail) {
```

Comment:
Does it make sense to move the 5 tests with 'if not edit email then return' check into their own file/suite called edit_email_test or something?